### ISSUE TITLE
Accessibility Audit: Conduct tests page user flows

### ISSUE THREAD

**Issue by mehansen** [CDCgov/prime-simplereport#5774]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the Conduct tests page user flows. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/app/queue`

Use cases to test:
* empty queue page
* single patient in queue
* multiple patients in queue
* test questionnaire modal
* adding test using patient search (inc search returned no results and multiple results)
* backdating a result
* multiplex and covid-only device
* test timer
* success toast (stretch: try an error toast as well?)

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

**Comment by BobanL** [comment]
No issues found
